[2024-01-14T08:22:06.238+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: jobinfo_jobplanet.create_infomation_s3 manual__2024-01-14T08:21:58.903828+00:00 [queued]>
[2024-01-14T08:22:06.245+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: jobinfo_jobplanet.create_infomation_s3 manual__2024-01-14T08:21:58.903828+00:00 [queued]>
[2024-01-14T08:22:06.246+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-01-14T08:22:06.246+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-01-14T08:22:06.246+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-01-14T08:22:06.261+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): create_infomation_s3> on 2024-01-14 08:21:58.903828+00:00
[2024-01-14T08:22:06.268+0000] {standard_task_runner.py:55} INFO - Started process 12332 to run task
[2024-01-14T08:22:06.270+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'jobinfo_jobplanet', 'create_infomation_s3', 'manual__2024-01-14T08:21:58.903828+00:00', '--job-id', '423', '--raw', '--subdir', 'DAGS_FOLDER/jobinfo_jobplanet.py', '--cfg-path', '/tmp/tmp71rkmiso']
[2024-01-14T08:22:06.271+0000] {standard_task_runner.py:83} INFO - Job 423: Subtask create_infomation_s3
[2024-01-14T08:22:06.647+0000] {task_command.py:388} INFO - Running <TaskInstance: jobinfo_jobplanet.create_infomation_s3 manual__2024-01-14T08:21:58.903828+00:00 [running]> on host 7b77d04f99d6
[2024-01-14T08:22:06.760+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=jobinfo_jobplanet
AIRFLOW_CTX_TASK_ID=create_infomation_s3
AIRFLOW_CTX_EXECUTION_DATE=2024-01-14T08:21:58.903828+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-14T08:21:58.903828+00:00
[2024-01-14T08:22:11.335+0000] {jobinfo_jobplanet.py:79} INFO - 11111111111111111
[2024-01-14T08:22:11.356+0000] {jobinfo_jobplanet.py:87} INFO - 2222222222222222222222222
[2024-01-14T08:22:11.479+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/jobinfo_jobplanet.py", line 116, in create_infomation_s3
    driver = jobplanet_login()
  File "/opt/airflow/dags/jobinfo_jobplanet.py", line 93, in jobplanet_login
    logging.info(id_input.get("driver.page_source"))
AttributeError: 'WebElement' object has no attribute 'get'
[2024-01-14T08:22:11.488+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=jobinfo_jobplanet, task_id=create_infomation_s3, execution_date=20240114T082158, start_date=20240114T082206, end_date=20240114T082211
[2024-01-14T08:22:11.502+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 423 for task create_infomation_s3 ('WebElement' object has no attribute 'get'; 12332)
[2024-01-14T08:22:11.537+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-01-14T08:22:11.699+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
